Mathlib/Topology/Algebra/Category/ProfiniteGrp.lean

```diff
@@ -161,4 +161,103 @@ instance : HasForget₂ ProfiniteGrp Grp where
     map := fun f => f.toMonoidHom
   }
 
+/--The functor mapping a profinite group to its underlying profinite space-/
+def profiniteGrpToProfinite : ProfiniteGrp ⥤ Profinite where
+  obj G := G.toProfinite
+  map f := ⟨f, by continuity⟩
+
+instance instProfiniteGrpToProfiniteFaithful : profiniteGrpToProfinite.Faithful := {
```

Comment:
This instance doesn't need a name if you follow another suggestion of mine below.